### PR TITLE
Travis CI added

### PR DESCRIPTION
**Pull request by keachi**
Change the Travis CI, should do the same as the GitLab CI does.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -16,8 +16,15 @@ before_script:
...
-  - for file in */tasks/*; do egrep -H '^- name:.*[[:upper:]]' "$file" && exit 1; done
+  - for dir in *; do
+      test -d "$dir" &&
+      echo "$dir" &&
+      ansible-lint -R -r .testenv/lint-rules/ "$dir";
+    done
+  - for dir in *; do
+      test -d "$dir" &&
+      echo "$dir" &&
+      yamllint -c ./.testenv/yamllint.yml "$dir";
```
Comment (karras) [inline .travis.yml]: `./.` and above you use `.testenv`, can this be aligned?

File: .travis.yml [hunk 2]
```diff
@@ -16,8 +16,15 @@ before_script:
   - yamllint --version
 
 script:
-  - for dir in *; do test -d "$dir" && echo "$dir" && ansible-lint "$dir"; done
-  - for dir in *; do test -d "$dir" && echo "$dir" && yamllint -c ./.testenv/yamllint.yml "$dir"; done
-  - for file in */tasks/*; do egrep -H '^- name:.*[[:upper:]]' "$file" && exit 1; done
```
Comment (karras) [inline .travis.yml:21]: We no longer check for uppercase words in the `name` sections?

**Review by karras — CHANGES_REQUESTED** [review]

File: .travis.yml (hunk 2 above)
Comment (keachi) [reply .travis.yml:21]: this is will be checked with ansible-lint

File: .travis.yml (hunk 1 above)
Comment (keachi) [reply .travis.yml]: done

**Review by karras — APPROVED** [review]

**Pull request merged**